week-1/2-mandatory/3-function-output.js

```diff
@@ -1,16 +1,19 @@
 // Add comments to explain what this function does. You're meant to use Google!
+//  This function is returning floating-point, pseudo-random number in the range 0 to less than 1. We will then make it a whole number by multiplying it by 10.
```

Comment:
Almost!  If you run the function you'll see the numbers coming back are not whole.